test/parallel_api/ranges/std_ranges_adjacent_find.pass.cpp

```diff
@@ -22,10 +22,12 @@ main()
     using namespace test_std_ranges;
     namespace dpl_ranges = oneapi::dpl::ranges;
 
-    test_range_algo<0>{big_sz}(dpl_ranges::adjacent_find, std::ranges::adjacent_find, binary_pred);
-    test_range_algo<1>{}(dpl_ranges::adjacent_find, std::ranges::adjacent_find, binary_pred, proj);
-    test_range_algo<2, P2>{}(dpl_ranges::adjacent_find, std::ranges::adjacent_find, binary_pred, &P2::x);
-    test_range_algo<3, P2>{}(dpl_ranges::adjacent_find, std::ranges::adjacent_find, binary_pred, &P2::proj);
+    auto adj_find_checker = TEST_PREPARE_CALLABLE(std::ranges::adjacent_find);
```

Comment:
Could you include `test_macros.h` for `TEST_PREPARE_CALLABLE` (here and in other tests)?